cve_bin_tool/cli.py

```diff
@@ -575,6 +581,12 @@ def main(argv=None):
     epss_percentile = 0
     if float(args["epss_percentile"]) > 0:
         epss_percentile = float(args["epss_percentile"]) / 100
+        LOGGER.critical(f"epss percentile stored {epss_percentile}")
+
+    epss_probability = 0
+    if float(args["epss_probability"]) > 0:
+        epss_probability = float(args["epss_probability"]) / 100
+        LOGGER.critical(f"epss percentile stored {epss_probability}")
```

Comment:
This should be LOGGER.debug (if at all) - we don't log the CVSS score for example